barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/client_ivc_recursive_verifier.cpp

```diff
@@ -0,0 +1,19 @@
+#include "client_ivc_recursive_verifier.hpp"
+
+namespace bb::stdlib::recursion::honk {
+
+void ClientIvcRecursiveVerifier_::verify(const ClientIVC::Proof& proof, VerifierInput& data)
+{
+    // WORKTODO: Perform Goblin recursive verification here
+
+    // Perform recursive folding verification
+    FoldingVerifier folding_verifier{ builder, data };
+    auto recursive_verifier_accumulator = folding_verifier.verify_folding_proof(proof.folding_proof);
+    auto native_verifier_acc = std::make_shared<VerifierInput::Instance>(recursive_verifier_accumulator->get_value());
```

Comment:
thinking out loud: if these two would share the same builder we wouldn't have to call get_value here. But probably not worth investing time atm, especially since we're not sure what's gonna happen when adding ZK

Reply:
Yeah not a top priority but I agree, might make sense to have these two more integrated. Currently we might be duplicating the final accumulator in the witness doing it this way